### PR TITLE
Add 'ros_gencpp', 'ros_geneus', 'ros_genlisp', 'ros_gennodejs', and 'ros_genpy' package

### PR DESCRIPTION
**Pull request by lsolanka**
A combined pull request for several ROS message generation packages. Hope this is fine instead of separate pull requests.

* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":
    * ros_gencpp
        * https://travis-ci.org/lsolanka/hunter/builds/370311482
        * https://ci.appveyor.com/project/lsolanka/hunter/build/1.0.23
    * ros_geneus
        * https://travis-ci.org/lsolanka/hunter/builds/370327960
        * https://ci.appveyor.com/project/lsolanka/hunter/build/1.0.26
    * ros_genlisp
        * https://travis-ci.org/lsolanka/hunter/builds/370328409
        * https://ci.appveyor.com/project/lsolanka/hunter/build/1.0.27
    * ros_gennodejs
        * https://travis-ci.org/lsolanka/hunter/builds/370328687
        * https://ci.appveyor.com/project/lsolanka/hunter/build/1.0.28
    * ros_genpy
        * https://travis-ci.org/lsolanka/hunter/builds/370348269
        * https://ci.appveyor.com/project/lsolanka/hunter/build/1.0.30

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull requests:
    * ros_gencpp => #230
    * ros_geneus =>  #231
    * ros_genlisp => #232
    * ros_gennodejs => #233
    * ros_genpy => #234

### CODE REVIEW

File: docs/packages/pkg/ros_gencpp.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+.. _pkg.ros_gencpp:
+
+ros_gencpp
+==========
+
+-  `Official <https://github.com/ros/gencpp>`__
+-  `Hunterized <https://github.com/hunter-packages/ros_gencpp>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/ros_gencpp/CMakeLists.txt>`__
+-  Added by
+    - `Lukas Solanka <https://github.com/lsolanka>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```
Comment (ruslo) [inline docs/packages/pkg/ros_gencpp.rst]: `s,N,1416,`

Comment (lsolanka) [reply docs/packages/pkg/ros_gencpp.rst]: Done, sorry for that.

**Comment by ruslo** [comment]
Testing,

ros_gencpp:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3760
* https://travis-ci.org/ingenue/hunter/builds/371224389?utm_source=github_status&utm_medium=notification

ros_geneus:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3761
* https://travis-ci.org/ingenue/hunter/builds/371224598?utm_source=github_status&utm_medium=notification

ros_genlisp:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3762
* https://travis-ci.org/ingenue/hunter/builds/371224853?utm_source=github_status&utm_medium=notification

ros_gennodejs:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3763
* https://travis-ci.org/ingenue/hunter/builds/371225166?utm_source=github_status&utm_medium=notification

ros_genpy:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3764
* https://travis-ci.org/ingenue/hunter/builds/371225377?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.20.55

**Pull request merged**